modules/administration-guide/partials/con_setup-backup-server.adoc

```diff
@@ -1,22 +1,34 @@
 [id="con_setup-backup-server"]
 = Setting up a backup server
 
-This section describes which backup servers supported in {prod-short} and gives hints on how to setup one.
+The following section describes the supported {prod-short} backup servers and provides information for their setup.
 
 include::partial$snip_internal-backup-server-warning.adoc[]
 
-{prod-short} uses link:https://restic.readthedocs.io/en/latest/[`restic`] tool to manage backup snapshots and to push / pull backup data to / from a backup server.
+{prod-short} uses the link:https://restic.readthedocs.io/en/latest/[`restic`] tool to: 
 
+* manage backup snapshots
+
+* push to or to pull backup data from a backup server
++
 [NOTE]
 ====
-`restic` is licensed under `BSD 2-Clause License`
+The `restic` backup tool is licensed under the link:https://opensource.org/licenses/BSD-2-Clause:[BSD 2-Clause] license.
 ====
 
-The following types of backup servers are supported in {prod-short} at the moment:
+.The current backup servers supported for {prod-short}:
+
+* `REST`
++
+** The `REST` server was a solution design for use with the `restic` tool. 
+** See link:https://restic.readthedocs.io/en/latest/[How to set up a REST server] documentation.
+
+* `Amazon S3` and API compatible alternatives.
+** See link:https://docs.aws.amazon.com/s3/index.html[AWS S3 Simple Storage Service Documentation] or the alternative documentation of your preference.
```

Comment:
> or the alternative documentation of your preference. 

Means alternative docs, but I meant docs of the alternative services (that have compatible API with AWS)